autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/ui/ExistingCredentialStoreInterrogatingMatchDetector.kt

```diff
@@ -19,13 +19,17 @@ package com.duckduckgo.autofill.ui
 import com.duckduckgo.autofill.store.AutofillStore
 import com.duckduckgo.di.scopes.AppScope
 import com.squareup.anvil.annotations.ContributesBinding
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
 import javax.inject.Inject
 
 @ContributesBinding(AppScope::class)
 class ExistingCredentialStoreInterrogatingMatchDetector @Inject constructor(private val autofillStore: AutofillStore) :
     ExistingCredentialMatchDetector {
 
     override suspend fun determine(currentUrl: String, username: String, password: String): AutofillStore.ContainsCredentialsResult {
-        return autofillStore.containsCredentials(currentUrl, username, password)
+        return withContext(Dispatchers.IO) {
```

Comment:
Why are we not using DispatcherProvider.io here?

Reply:
Laziness 😊

Will switch to that.